content/guides/core-concepts/retry-ability.md

```diff
@@ -33,9 +33,19 @@ There are two types of methods you can call in your Cypress tests: **commands**
 and **assertions**. For example, there are 6 commands and 2 assertions in the
 test below.
 
-```javascript
+:::cypress-visit-mount-test-example
```

Comment:
I'm not sure it makes sense to do a tabbed example here because the screenshot below can only show either CT or E2E. Maybe it makes sense to just choose one.

Reply:
Yeah. I appreciated having the alternate context as part of the section discussing `Commands vs assertions`. We can probably just show one. 

Or, tabbed screenshot UI! 🤦‍♂️